drivers/sensor/lps28dfw/lps28dfw.c

```diff
@@ -0,0 +1,332 @@
+/* ST Microelectronics LPS28DFW pressure and temperature sensor
+ *
+ * Copyright (c) 2023 STMicroelectronics
+ * Copyright (c) 2023 PHYTEC Messtechnik GmbH
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Datasheet:
+ * https://www.st.com/resource/en/datasheet/lps28dfw.pdf
+ */
+
+#define DT_DRV_COMPAT st_lps28dfw
+
+#include <zephyr/drivers/sensor.h>
+#include <zephyr/kernel.h>
+#include <zephyr/device.h>
+#include <zephyr/init.h>
+#include <zephyr/sys/byteorder.h>
+#include <zephyr/sys/__assert.h>
+#include <zephyr/logging/log.h>
+
+#include "lps28dfw.h"
+
+#define LPS28DFW_SWRESET_WAIT_TIME	50
```

Comment:
`LPS28DFW_SWRESET_WAIT_TIME_US`

Reply:
This was taken from lps22df, just merged. Probably we missed it during the review process.
I'm proposing to extend lps22df driver with lps28dfw stuff, which are not that much.

If agreed, we can go in this direction and @jremmert-phytec-iot may also follow up with these requests in lps22df driver.

Reply:
yeah sorry, looks like I missed it